Backend-Roadmap/Context/AppDbContext.cs

```diff
@@ -13,7 +13,7 @@ public AppDbContext(DbContextOptions<AppDbContext> options) : base(options)
 
         public DbSet<Users> Users { get; set; }
         public DbSet<Project> Project { get; set; }
-        public DbSet<Task> Task { get; set; }
+        public DbSet<Task> Tasks { get; set; }
```

Comment:
It's not a good ideia to use C# Keywords in your code like for example: sealed, continue, abstract
Here you are creating a table Tasks which is DbSet<Task> . Maybe you will have trouble with it.
Since you pluralized the name DbSet<Task> Tasks   to Tasks maybe you will not have problems.
But a reminder: don't use C# reserved words on your code